apps/etterlatte-beregning/src/main/kotlin/avkorting/Avkorting.kt

```diff
@@ -459,7 +459,7 @@ enum class OverstyrtInnvilgaMaanederAarsak {
 
 data class Aarsoppgjoer(
     val id: UUID,
-    val aar: Int,
+    val aar: Int, // TODO validere på format?
```

Comment:
i en init?

Reply:
Eller klasse som `sakId` som validerer når verdi settes. Nå kan vi ikke garantere at `aar` er 2024, det kan være 24, eller kun 2 for den del. Vi starter å få en del sjekker som sammenligner `aar == YearMonth.year`. Greit å vite at de alltid har samme utgangspunkt.